corehq/apps/data_interfaces/forms.py

```diff
@@ -603,6 +648,14 @@ def save_criteria(self, rule):
                 criteria.definition = definition
                 criteria.save()
 
+            for item in self.cleaned_data['ucr_filter_definitions']:
+                definition = UCRFilterDefinition.objects.create(
```

Comment:
Does this need to handle duplicate filters?

Reply:
Can you elaborate? Do you mean the case where a user enters the same `configured_filter` JSON twice?

Reply:
> Do you mean the case where a user enters the same configured_filter JSON twice?

Yes, that's what I meant!